test/image_processing/lanczos_scaling.cpp

```diff
@@ -0,0 +1,82 @@
+//#include <boost/gil/image_processing/scaling.hpp>
+
+#include <boost/gil/extension/io/png.hpp>
+#include <boost/gil/extension/io/jpeg.hpp>
+#include <boost/gil/io/read_image.hpp>
+#include <boost/gil/io/write_view.hpp>
+
+#include <boost/gil/image.hpp>
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/rgb.hpp>
+#include <boost/gil/pixel.hpp>
+
+#include <boost/mpl/int.hpp>
+
+#include <cmath>
+#include <iostream>
+
+namespace gil = boost::gil;
+namespace mpl = boost::mpl;
+using image_view = decltype(gil::view(std::declval<gil::rgb8_image_t&>()));
+double sinc(double x) {
+    constexpr double pi = 3.14;
+    return std::sin(x * pi) / (x * pi);
+}
+
+double lanczos(long int x, long int a) {
```

Comment:
I assume `lanczos`, `lanczos_at`, and `scale_lanczos` are the three new functions you are proposing (and which the current file is supposed to be testing, right ? Shouldn't the functions themselves then be defined elsewhere, to be usable by `Boost.GIL` users ?

Reply:
In either case, these functions need (inline) documentation. As we are using doxygen, I'd suggest simple doxygen-style comments such as
```
//! bla bla bla
```
potentially with inline markdown markup. A simple description of what the function is doing, and, if necessary, what the input arguments and return value mean.

Reply:
I believe commenting is the only thing left from the first round of review from you